backend/cms/templates/pages/tree.html

```diff
@@ -54,26 +63,31 @@ <h2 class="heading font-normal mb-2">{% trans 'Pages' %}</h2>
             <tr class="border-b border-solid border-grey-light">
                 <th class="text-sm text-left uppercase py-3 pl-2 pr-4"></th>
                 <th class="text-sm text-left uppercase py-3 pl-4">{% trans 'ID' %}</th>
-                <th class="text-sm text-left uppercase py-3 px-2">{% trans 'Title' %}</th>
+                <th class="text-sm text-left uppercase py-3 pl-2">{% trans 'Version' %}</th>
+                <th class="text-sm text-left uppercase py-3 px-2">{% trans 'Title in' %} {{ language.translated_name }}</th>
+                {% get_current_language as LANGUAGE_CODE %}
+                {% unify_laguage_code LANGUAGE_CODE as LANGUAGE_CODE %}
+                {% if LANGUAGE_CODE != language.code %}
+                    <th class="text-sm text-left uppercase py-3 px-2">{% trans 'Title in' %} {% translated_language_name LANGUAGE_CODE %}</th>
+                {% endif %}
                 <th class="text-sm text-left uppercase py-3 px-2">
                     <div class="lang-grid flags whitespace-no-wrap">
-	                    {% for lang in languages %}
-		                    <a href="{% url 'pages' region_slug=region.slug language_code=lang.code %}">
-			                    <img src="{% static '' %}images/flags/{{ lang.code }}.png" title="{{ lang.name }}" />
-		                    </a>
-	                    {% endfor %}
+                        {% for lang in languages %}
+                            <a href="{% url 'pages' region_slug=region.slug language_code=lang.code %}">
+                                <img src="{% static '' %}images/flags/{{ lang.code }}.png" title="{{ lang.name }}" />
```

Comment:
```suggestion
                                <img src="{% static '' %}images/flags/{{ lang.code }}.png" title="{{ lang.name }}" />
                                <span title="{{ lang.translated_name }}">
```

Reply:
```suggestion
                                <span title="{{ lang.translated_name }}">
                                <img src="{% static '' %}images/flags/{{ lang.code }}.png" />
                                />
                                
                                
```

Reply:
Thanks, I updated my commit with the working tooltip.